src/atoms/Text/Text.native.js

```diff
@@ -0,0 +1,43 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+import theme from '../../tokens/theme';
+import { Text as NativeText } from 'react-native';
+import { flattenArray } from '../../utils/index';
+
+const fontFamilies = {
```

Comment:
Should we move this to a different file `fontFamilies`?

Reply:
We can but it will not be used anywhere else